relion/viewers/viewer_base.py

```diff
@@ -587,10 +585,17 @@ def _createAngDistChimera(self, it):
                 self.createAngDistributionSqlite(
                     sqliteFn, nparts,
                     itemDataIterator=self._iterAngles(mdOut))
-
-            return ChimeraClientView(volFn,
-                                     angularDistFile=sqliteFn,
-                                     spheresDistance=radius)
+            if hasattr(self.protocol, 'outputVolumes'):
+                vol = self.protocol.outputVolumes.getFirstItem()
```

Comment:
@pconesa is it always selecting origin if the first class even if I'm displaying e.g. class 3?

Reply:
Yes, we discussed that when making the changes and we thought that origin should be the same for all classes.